tasks/fix-cat2.yml

```diff
@@ -1941,15 +1941,15 @@
       - RHEL-07-040670
 
 - name: "MEDIUM | RHEL-07-040680 | PATCH | The system must be configured to prevent unrestricted mail relaying."
-  command: "true"
-  changed_when: no
-  when: rhel_07_040680
+  command: "postconf -e 'smtpd_client_restrictions=permit_mynetworks, reject'"
```

Comment:
Can we assume postfix is installed?  It is by default, but it's a leaf package, so can be easily removed.

Reply:
Probably shouldn't assume it, no. Since the STIG says to check if it's installed, and if it's not, 040680 is not applicable.